runtime/queries/cpp/highlights.scm

```diff
@@ -34,6 +34,10 @@
 (auto) @type
 "decltype" @type
 
+(ref_qualifier (["&" "&&"]) @type.builtin)
+(reference_declarator (["&" "&&"]) @type.builtin)
+(abstract_reference_declarator (["&" "&&"]) @type.builtin)
```

Comment:
Very minor nit but the parens are not needed here around the `[..]`:

```scm
(ref_qualifier ["&" "&&"] @type.builtin)
```

is equivalent

Reply:
Thanks! I'm definitely having trouble internalizing exactly when I need them...